doc/howtos/rdtraining/16_guidelines_pr.rst

```diff
@@ -0,0 +1,154 @@
+
+Example: The branch master-sale-fixes-abc on odoo-dev/odoo is a branch containing fixes for the
+sales app in the odoo/odoo repository, to be deployed in master and done by ABC.
+
+Commit your code:
+
+
+.. code-block:: console
+
+    $ git add .
+    $ git commit
+
+
+**Everyone is reading your commit messages!**
+
+The commit message is very important, follow the :ref:`Developer guidelines <reference/guidelines/git>`.
+
+
+Push your new branch to your developement repository:
+
+.. code-block:: console
+
+    $ git push -u master-my_first_branch-xyz
+
+.. warning:: NEVER use `git push --force` (or `git push -f`) in a branch that does not belong to you
+             nor in a stable branch. Never, never, never!
+
+.. tip:: Remember that you can use `git status` at any step to know the current status of your branch
+.. tip:: You can create git alias to shorten some of theses commands. Ask your colleagues around you
+         if they use git aliases and get inspired by their work.
+
+Create your first PR
+--------------------
+
+After pushing your branch to your development repository, the output will look like this:
+
+.. code-block:: console
+
+    Enumerating objects: 24, done.
+    Counting objects: 100% (24/24), done.
+    Compressing objects: 100% (20/20), done.
+    Writing objects: 100% (23/23), 6.67 KiB | 525.00 KiB/s, done.
+    Total 23 (delta 1), reused 0 (delta 0)
+    remote: Resolving deltas: 100% (1/1), done.
+    remote:
+    remote: Create a pull request for 'master-my_first_branch-xyz' on GitHub by visiting:
+    remote:      https://github.com/xyz-odoo/technical-training-sandbox/pull/new/master-my_first_branch-xyz
+    remote:
+    To github.com:xyz-odoo/technical-training-sandbox.git
+    * [new branch]      master-my_first_branch-xyz -> master-my_first_branch-xyz
+    Branch 'master-my_first_branch-xyz' set up to track remote branch 'master-my_first_branch-xyz' from 'training-dev'.
+
+There are two ways to create the PR:
+
+- Click on the link displayed in the output of the push command.
+- Open a browser in your development repository `https://github.com/xyz-odoo/technical-training-sandbox`.
+  A button will suggest you to create a new pull request.
+
+  .. image:: 16_guidelines_pr/media/pr_from_branch.png
+
+You will notice that your commit message is used as pull request message, this occurs if you have only 1 commit.
+If you have multiple commits, you can make a summary of the commit as PR message. Or, if you have few
+commits, just copy/paste your commit messagges as the PR message.
+
+.. image:: 16_guidelines_pr/media/pr_message.png
+
+
+Test on the runbot
+==================
+
+Odoo has its own :abbr:`CI (Continuous integration)` server named `runbot <https://runbot.odoo.com/>`__. All
+the commits, branches and PR will be tested to avoid regressions or breaking the stable versions.
+All the runs that pass the tests are deployed on their own server with demo data.
```

Comment:
Mebbe @d-fence could setup a runbot project for the repo now that multi-project runbot works?